flow/designs/sky130hd/chameleon_hier/config.mk

```diff
@@ -43,16 +43,18 @@ export VERILOG_FILES = \
 export HAS_IO_CONSTRAINTS = 1
 export ENABLE_DPO = 0
 export MACRO_PLACE_CHANNEL  = 160 160
-export MACRO_PLACE_HALO = 2 2
+export MACRO_PLACE_HALO = 160 160
```

Comment:
Why is such a large halo needed?

Reply:
Less than that causing horizontal and vertical congestion for for apb_xxx_inst0 alone.

Reply:
If I use halo 120 and above no congestion occurs. So I kept halo and channel the same. @maliberty